### ISSUE TITLE
Create application main layout

### ISSUE THREAD

**Issue by brauliodiez** [Lemoncode/quickmock#6]
** Pending on accepting invitation, assigned to: @LourdesRsdp

We have to define a wireframe layout to position:
  - Toolbar.
  - Right hand sidebar.
  - Main canvas.

(*) in the future we will position properties panel but we can put this right now on hold.

Please talk with Leticia about this she will provide you with the the entry point.

<img width="1169" alt="image" src="https://github.com/Lemoncode/quickmock/assets/1457912/44f791aa-78e7-4461-bf65-2764dc20b5e5">
